azure/v11.4.0/README.md

```diff
@@ -0,0 +1,58 @@
+## :zap:  Giant Swarm Release 11.4.0 for Azure :zap:
+
+**If you are upgrading from 11.3.3, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+In this release NGINX Ingress Controller LoadBalancer Service external traffic policy has been made configurable, and the policy default has been changed from `Cluster` to `Local`. These changes:
+
+- Enable source IP preservation, needed for IP based security access control.
+- Remove unnecessary ingress traffic intra-cluster network hops, increasing concurrent connection capacity and reducing latency.
+- Allow configuration to be reverted, where these defaults and associated tradeoffs are found not to be appropriate.
+
+Management of NGINX IC LoadBalancer Service is moved from azure-operator to NGINX IC App itself to:
+
+- Enable external traffic policy configurability, in a way consistent with other NGINX IC configuration options.
+- Lay the foundation for making NGINX IC App optional and not pre-installed in a future Azure platform release.
+
+When upgrading existing clusters from older Azure platform releases, migration of LoadBalancer Service is automated but not fully automatic. Therefore we ask of you to delegate cluster upgrades to your SE.
+
+**Note for SEs:** After cluster upgrade to 11.4.0, both old `ingress-loadbalancer` LoadBalancer Service managed by azure-operator and new one `nginx-ingress-controller` managed by NGINX IC App remain on the cluster. To switch the ingress traffic to the new LoadBalancer and remove old NGINX LoadBalancer Service without downtime please:
+
+- Together with the customer have any firewall in front of NGINX reconfigured to allow both old and new LoadBalancer Service IPs.
+- Next use [the migration script](https://github.com/giantswarm/azure-operator/blob/master/scripts/migrate-nginx-ingress-controller.sh) to switch DNS records to the new load balancer IP. Now delete the old NGINX IC LoadBalancer Service.
```

Comment:
Why don't we specify to wait some time before we delete the old load balancer object?

Reply:
```suggestion
- Next use [the migration script](https://github.com/giantswarm/azure-operator/blob/master/scripts/migrate-nginx-ingress-controller.sh) to switch DNS records to the new load balancer IP. The script ensures IP is assigned to the new LB, and also that the cluster DNS records resolve to it instead of old IP.
- Now delete the old NGINX IC LoadBalancer Service.
```
Initial step (firewall configuration) is not possible without new LB having IP assigned.
Migration script already waits for IP to be assigned and also DNS to resolve to it. I've added a note about that script behavior. In the steps list I've split the two steps (script running and old LB deletion). Please propose any additional change you have in mind.